site/content/docs/user-guide/setup/configuration.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Configuration
+slug: configuration
+weight: 300
+---
+
+### Disable update checks
+
+Krew will occasionally check if a new version is available to remind you to
+upgrade to a newer version.
+
+This is done by calling the GitHub API, and we do not collect any data from your
+machine during this process.
+
+If you want to disable the update checks, set the `KREW_NO_UPGRADE_CHECK`
+environment variable. To permanently disable this, add the following to your
+`~/.bashrc`, `~/.bash_profile`, or `~/.zshrc`:
+
+```shell
+export KREW_NO_UPGRADE_CHECK=1
+```
+
+### Use a different default index
+
+By default Krew uses [krew-index][ki] as the default plugin index. This is the
```

Comment:
I'd rewrite this paragraph as follows:


> When Krew is installed, it automatically initializes an index named `default` pointing to [krew-index][ki] repository. If you choose to remove this index, see [custom indexes] documentation. You can also force Krew to point it to another repository by setting `KREW_DEFAULT_INDEX_URI`.

+mash that up with existing links + next paragraph you have.

Reply:
What about:
 
> When Krew is installed, it automatically initializes an index named `default`
pointing to the [krew-index][ki] repository. You can force Krew to use a
different repository by setting `KREW_DEFAULT_INDEX_URI` before running the
[installation instructions]({{<ref "install.md">}}) or after [removing the
default index]({{<ref "using-custom-indexes.md">}}).

> To use a different default index, set the `KREW_DEFAULT_INDEX_URI` environment
variable in your `~/.bashrc`, `~/.bash_profile`, or `~/.zshrc`:

I think it would make sense to keep the parts on when the env var needs to be set together instead of mentioning that you can set it after deleting the default index in the first paragraph and before krew installation in the second paragraph.

Also should I make this `draft: true`? IIRC the docs get published automatically.

Reply:
Don't worry about setting it as draft without a release being available. I highly suspect anyone’ll come looking for it.

I think your edit is good, make sure to mention this URI needs to point to a git repository reference (can be a real url, or another git-remote protocol like foo://bar/quux.git).

Reply:
Made the change, @ahmetb mind taking another look?